py/selenium/webdriver/common/proxy.py

```diff
@@ -54,10 +54,8 @@ def load(cls, value):
         value = str(value).upper()
         for attr in dir(cls):
             attr_value = getattr(cls, attr)
-            if isinstance(attr_value, dict) and \
-                    'string' in attr_value and \
-                    attr_value['string'] is not None and \
-                    attr_value['string'] == value:
+            # `attr_value['string'] is not None` probably not required as `attr_value['string'] == value`
+            if isinstance(attr_value, dict) and 'string' in attr_value and attr_value['string'] == value:
```

Comment:
# `attr_value['string'] is not None` probably not required as `attr_value['string'] == value` check is already being done

Reply:
We don't need the comment  here as it adds little to no value